### ISSUE TITLE
Design request

### ISSUE THREAD

**Issue by shinze** [arrowtype/recursive#334]
The apostrophe seems huge compared to the ´of é and from a different font design. 
<img width="553" alt="image" src="https://user-images.githubusercontent.com/218396/78371246-fc4bf700-75c7-11ea-9dab-b8734f1559df.png">

Here is what the same words are looking like with IBM Plex font for illustration.
<img width="568" alt="image" src="https://user-images.githubusercontent.com/218396/78371477-5056db80-75c8-11ea-9ab2-257050748b03.png">

**Comment by raphaelbastide** [comment]
@arrowtype Are you saying this design choice (to keep bolder punctuations) inherits from the monospace nature of the font, even in the proportional version? I am currently setting a text with `"MONO" 0, "CASL" 0, "wght" 370, "slnt" 0, "CRSV" 0.5` and I notice the gray sometimes jumps because of the weight of `’`. To be honest I don’t know yet if I like it or not, but I was curious why you choose that direction.

![Screenshot_2021-04-23_11-41-01](https://user-images.githubusercontent.com/193122/115853202-34b7b080-a429-11eb-9e81-95406113de57.png)

**Comment by arrowtype** [comment]
Yes, it was a conscious decision to keep larger-than-usual punctuation in the sans styles. I think a teacher of mine once said something like, "no one ever died because there was too obvious a difference between a period and a comma, but I bet the reverse is true."

However, you make a good case that perhaps the quote marks are a little too intense in the Sans styles. I will consider making them match the text color more closely in a future release.

**Comment by sipec** [comment]
I actually like it as it is because I use the proportional version of Recursive for programming
